### PR TITLE
Add docker test in MicroOS

### PR DESCRIPTION
**Pull request by drpaneas**
MicroOS is an operating system optimized for running the next
generation of applications with Linux containers. Thus, running
a container-based test (e.g. docker) is mandatory.

### CODE REVIEW

**Comment by drpaneas** [comment]
test: http://ultron.suse.de/tests/602#

File: tests/console/docker.pm [hunk 1]
```diff
@@ -28,8 +28,13 @@ use strict;
 sub run {
     select_console("root-console");
 
-    # install the docker package
-    zypper_call("in docker");
+    if (check_var("DISTRI", "caasp")) {
```
Comment (sysrich) [inline tests/console/docker.pm]: -1 - You need to test this on Kubic as well as CaaSP :)

Please either check for caasp || kubic or create a "is_casp" sub in the main.pm so you can use it here and we can reuse it when this problem comes up

Comment (mitiao) [reply tests/console/docker.pm]: FYI https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L264

**Comment by drpaneas** [comment]
I made the changes @sysrich  could you have a look on them?

**Pull request merged**